src/Packetery/Module/Checkout.php

```diff
@@ -1038,4 +1053,16 @@ private function getTaxClassWithMaxRate() {
 		return $resultTaxClass;
 	}
 
+	/**
+	 * Tells cart price with VAT and without Coupons.
+	 *
+	 * @return float
+	 */
+	private function getCartPriceIncludingVatAndCoupons():float {
```

Comment:
V docu je without, v názvu including. Nemělo by to být v rozporu.

Reply:
getCartPriceIncludingVatWithoutCoupons je lepsi?

Reply:
Ano.